test/cuNDArray_blas_test.cpp

```diff
@@ -7,150 +7,235 @@
 using namespace Gadgetron;
 using testing::Types;
 
-template <typename T> class cuNDArray_blas_Real : public ::testing::Test 
-{
-protected:
-  virtual void SetUp() {
-    size_t vdims[] = {37, 49, 23, 19}; //Using prime numbers for setup because they are messy
-    dims = std::vector<size_t>(vdims,vdims+sizeof(vdims)/sizeof(size_t));
-    Array = cuNDArray<T>(&dims);
-    Array2 = cuNDArray<T>(&dims);
-  }
-  std::vector<size_t> dims;
-  cuNDArray<T> Array;
-  cuNDArray<T> Array2;
+template <typename T> class cuNDArray_blas_Real : public ::testing::Test {
+  protected:
+    virtual void SetUp() {
+        size_t vdims[] = {37, 49, 23, 19}; // Using prime numbers for setup because they are messy
+
+        dims = std::vector<size_t>(vdims, vdims + sizeof(vdims) / sizeof(size_t));
```

Comment:
Couldn't this just be:

```cpp
dims = std::vector<size_t>{37, 49, 23, 19};
```

Consolidating the line above into this. Same goes for below.